docs/deployment/pull-backup.rst

```diff
@@ -68,10 +75,13 @@ create the backup, retaining the original paths, excluding the repository:
 
 ::
 
-    borg create --exclude /borgrepo /borgrepo::archive /
+    borg create --exclude /borgrepo --files-cache ctime,size /borgrepo::archive /
 
-(For the sake of simplicity only ``/borgrepo`` is excluded here. You may want to
-set up an exclude file with additional files and folders to be excluded.)
+For the sake of simplicity only ``/borgrepo`` is excluded here. You may want to
+set up an exclude file with additional files and folders to be excluded. Also
+note that we have to modify Borg's file change detection behaviour – SSHFS does
+not support real inode numbers, so we have to supply the ``--files-cache``
```

Comment:
it doesn't matter whether they are "real", but rather whether they are stable.

so, they are not stable?